### PR TITLE
Add vagrant-hostmanager support for automatically configuring /etc/hosts

### PR DESCRIPTION
**Pull request by barnabycourt**
Configure /etc/hosts on both the host and guest boxes automatically if the vagrant-hostmanager plugin is installed.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -128,6 +128,13 @@ module Forklift
 
 
   Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    if Vagrant.has_plugin?("vagrant-hostmanager")
+      config.hostmanager.enabled = true
+      config.hostmanager.manage_host = true
```
Comment (ekohl) [inline Vagrantfile]: Personally I don't use vagrant but I don't know if this is desired by most.

Comment (stbenjam) [inline Vagrantfile]: If they have the plugin installed, I'd guess it's probably wanted.

Comment (thomasmckay) [inline Vagrantfile]: +1 useful plugin

Comment (stbenjam) [inline Vagrantfile]: If anyone wants just the client piece and not manage_host, we can always add some user configuration later.  For now I think these defaults are fine

Comment (ekohl) [inline Vagrantfile]: Agreed

**Comment by ekohl** [comment]
ACK

**Comment by ehelms** [comment]
Thanks @barnabycourt !

**Pull request merged**